models/services.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(sequelize, DataTypes) {
+    var Service = sequelize.define("Service", {
```

Comment:
Let's pluralize both instances of `Service` to `Services` to match Sequelize grammar